.github/workflows/openmldb-docker.yml

```diff
@@ -80,6 +76,5 @@ jobs:
           push: ${{ env.PUSH }}
           platforms: linux/amd64
           tags: |
-            ${{ env.GHCR_TAG }}
```

Comment:
why rm ghcr ?

Reply:
@aceforeverd we have decided to make dockerhub as the only place to download docker images, except that complication image required by the cicd (considering the download speed)